From 285dd667d6f508da1f862a2ca49986b843df8458 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Wed, 9 Sep 2020 20:01:08 -0400
Subject: [PATCH 1601/1936] Add ranges to effect parameters

Allows for the automation of additional effect parameters which
previously failed due to the automation subsystem not being able
to identify the range of each parameter.
---
 src/Effects/Effect.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Effects/Effect.h b/src/Effects/Effect.h
index e890422a..c61bab89 100644
--- a/src/Effects/Effect.h
+++ b/src/Effects/Effect.h
@@ -23,7 +23,7 @@
 #ifndef rEffPar
 #define rEffPar(name, idx, ...) \
   {STRINGIFY(name) "::i",  rProp(parameter) rDefaultDepends(preset) \
-   DOC(__VA_ARGS__), NULL, rEffParCb(idx)}
+      rLinear(0,127) DOC(__VA_ARGS__), NULL, rEffParCb(idx)}
 #define rEffParOpt(name, idx, ...) \
   {STRINGIFY(name) "::i:c:S",  rProp(parameter) rDefaultDepends(preset) \
    rProp(enumerated) DOC(__VA_ARGS__), NULL, \
-- 
2.47.0

